### PR TITLE
travis runtime

### PR DESCRIPTION
**Pull request by PowerfulBacon**
## About The Pull Request

travis runtime

## Why It's Good For The Game

travis runtime

## Changelog
:cl:
fix:travis runtime
/:cl:

### CODE REVIEW

**Comment by PowerfulBacon** [comment]
travis runtime

File: code/game/objects/structures/tables_racks.dm [hunk 1]
```diff
@@ -37,9 +37,11 @@
 
 /obj/structure/table/Bumped(mob/living/carbon/human/H)
 	. = ..()
+	if(!istype(H))
+		return
 	var/feetCover = (H.wear_suit && (H.wear_suit.body_parts_covered & FEET)) || (H.w_uniform && (H.w_uniform.body_parts_covered & FEET))
 	if(!istype(H) || H.shoes || feetCover || !(H.mobility_flags & MOBILITY_STAND) || HAS_TRAIT(H, TRAIT_PIERCEIMMUNE) || H.m_intent == MOVE_INTENT_WALK)
```
Comment (qwertyquerty) [inline code/game/objects/structures/tables_racks.dm]: Redundant check

Comment (PowerfulBacon) [reply code/game/objects/structures/tables_racks.dm]: oh I forgot to remove the second one

Comment (PowerfulBacon) [inline code/game/objects/structures/tables_racks.dm]:
```suggestion
	if(H.shoes || feetCover || !(H.mobility_flags & MOBILITY_STAND) || HAS_TRAIT(H, TRAIT_PIERCEIMMUNE) || H.m_intent == MOVE_INTENT_WALK)
```

**Review by zeskorion — APPROVED** [review]

**Pull request merged**